cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -470,6 +470,24 @@
     </description>
   </property>
 
+  <property>
+    <name>run.record.counter.entry.retention.seconds</name>
+    <value>3600</value>
+    <description>
+      Maximum amount of time (in seconds) in which a run record counter entry is retained.
```

Comment:
nit: adding a comment mentioning that if this config is updated, the interval config should be updated accordingly. Same comment in the other config